main.js

```diff
@@ -48,6 +49,13 @@ function setup() {
 		button.mousePressed(() => {
 			currentMode = modes[i]
 			currentMode.init()
+			if (file_selector.selectedIndex !== 0 && currentMode.name !== '☰') {
+				currentMode.overridePhrases(phraseCollectionList[file_selector.selectedIndex])
+			} else if (currentMode.name !== '☰') {
```

Comment:
Oh, there's so much wrong with this....
1. assuming that `currentMode` is of `Game` type simply if its name isn't something
2. manipulating the intimate properties of the game that may or may not be out of sync with how the game handles itself.
3. the game already knows how to use it's default set of phrases!

Reply:
Rather than trying to assume the type of `currentMode` and reaching into its inner workings to manipulate things, we should let the various different modes (should really be called "screens" now rather than "modes") reach out to get the information they need.  So you make a central repository that holds overriding phrases (if any) and the `Game` class or whoever is interested can query this repository for any overriding set of phrases for it to use instead of the default.

Reply:
Remind me to do lessons on Separation-of-concerns, encapsulation, inversion-of-control

Reply:
I think I've done what was asked and moved the check into a method called by the init. Otherwise I have no clue what exactly you're asking as your response is worded in a very confusing and roundabout way.

Reply:
Be specific. Which parts are confusing?  If it's the phrases repository then think of it this way:. Day we have a "phrases Oracle" whose purpose is to serve up phases to those who ask for them.  This Oracle gets it's data via importing or whatever means.  A game can ask the Oracle for a set of overriding phrases (if any), otherwise it will use it's default set of phrases.

This way, we make things more modular and independent.